src/Extraload/Extractor/Doctrine/QueryExtractor.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Extraload\Extractor\Doctrine;
+
+use Doctrine\DBAL\Connection;
+use Extraload\Extractor\ExtractorInterface;
+
+class QueryExtractor implements ExtractorInterface
```

Comment:
One of this two classes is redundant. I'd refactor it to only `QueryExtractor`.

`(Connection $conn, string $sql, array $values)` will do the same job if `$values = []`.

Reply:
I believe my original suggestion brings a bit of awareness to the API user about the fact that they are using either queries or prepared queries. Having those two is perhaps a way to separate concerns because the logic being run under the hood is a bit different. `QueryExtractor` runs a `$conn->query($sql);` under the hood, whereas the `PreparedQueryExtractor` runs a `$conn->prepare($sql);`.